common/common.go

```diff
@@ -124,3 +118,28 @@ func (e *ExecutionData) Clean() error {
 	e.Procedure = strings.ToLower(e.Procedure)
 	return nil
 }
+
+// NetworkParameters are network level configurations that can be
+// evolved over the lifetime of a network.
+type NetworkParameters struct {
+	// MaxBlockSize is the maximum size of a block in bytes.
+	MaxBlockSize int64
+	// JoinExpiry is the number of blocks after which the validators
+	// join request expires if not approved.
+	JoinExpiry int64
+	// VoteExpiry is the default number of blocks after which the validators
+	// vote expires if not approved.
+	VoteExpiry int64
+	// DisabledGasCosts dictates whether gas costs are disabled.
+	DisabledGasCosts bool
+}
```

Comment:
@charithabandi does this seem complete for our currently-used parameters?  Anything we don't use presently can be part of a scheme upgrade.

Reply:
Yeah, this looks good.